test/hamiltonian_consistency.jl

```diff
@@ -1,109 +1,112 @@
-using Test
-using DFTK
-using Logging
-import DFTK: mpi_sum
+@testsetup module HamConsistency
```

Comment:
I would vote for *not* indenting the `@testsetup` blocks. That probably makes the code less deeply nested, which overall leads to better readability. What do you think ?